envoy.bazelrc

```diff
@@ -17,6 +17,9 @@ build --host_javabase=@bazel_tools//tools/jdk:remote_jdk11
 build --javabase=@bazel_tools//tools/jdk:remote_jdk11
 build --enable_platform_specific_config
 
+# TODO(lambdai): wondering if wee8 only see envoy.bazelrc
+build --action_env=CXXFLAGS=-Wno-unused-variable
```

Comment:
`CXXFLAGS` is overwritten below in `libc++` and `rbe-toolchain-clang-libc++` configs, so you need to add this flag there as well:
```
build:libc++ --action_env=CXXFLAGS=-stdlib=libc++
```

Reply:
THanks!

Is that overwrittern over append? That means all touched config must introduce `-Wno-unused-variable`

Reply:
That's not going to work, please try what I suggested earlier, i.e. change only `envoy.bazelrc`.

Current:
```
build:libc++ --action_env=CXXFLAGS=-stdlib=libc++
build:rbe-toolchain-clang-libc++ --action_env=CXXFLAGS=-stdlib=libc++
```

Should be:
```
build --action_env=CXXFLAGS="-Wno-unused-variable"
build:libc++ --action_env=CXXFLAGS="-stdlib=libc++ -Wno-unused-variable"
build:rbe-toolchain-clang-libc++ --action_env=CXXFLAGS="-stdlib=libc++ -Wno-unused-variable"
```

Reply:
Sorry I am exploring at local. It looks like .bazelrc works but I am not sure.

Not using envoy.bazelrc because I want to keep envoy.bazel the same as envoy upstream

Reply:
It worked!